builders/deploy-manage/dapp/deploy.md

```diff
@@ -0,0 +1,183 @@
+
+Your runtime must implement the following:
+
+- The Cumulus SDK, as outlined in the [Base Setup to Connect to Polkadot](/builders/build/templates/base-setup-to-polkadot){target=_blank} section of the [Templates](/builders/build/templates){target=_blank} page
+- Tanssi modules for block production, as outlined in the [Base Setup to Support the Tanssi Protocol](/builders/build/templates/#base-setup-supporting-tanssi){target=_blank} section of the [Templates](/builders/build/templates){target=_blank} page
+
+You can upload your custom raw specification file by selecting the **Custom** template and adding your JSON specification file.
+
+![Upload a custom raw specification file to the Tanssi dApp.](/images/builders/deploy-manage/dapp/deploy/deploy-6.png)
+
+## Reserve your Appchain ID {: #reserve-appchain-id }
+
+If you haven't already done so, you'll need to reserve your Appchain ID, which will be required to register your Appchain on the relay chain and function within the Polkadot ecosystem.
+
+To reserve your Appchain ID, you'll need to submit a transaction. Please make sure to use the account you plan to launch your Appchain with when submitting the transaction.
+
+1. To initiate the transaction, click on **Reserve Appchain ID**
+2. Your wallet will pop up, and you'll need to submit the transaction
+
+![Reserve your Appchain ID via the Tanssi dApp.](/images/builders/deploy-manage/dapp/deploy/deploy-7.png)
+
+Once the transaction has successfully gone through, your Appchain ID will be displayed on the dApp, and you'll be able to click **Continue** to proceed to the next step. You'll notice that on your Alphanet relay chain account, 20 UNIT tokens have been removed from your transferrable balance and are now reserved.
+
+![Successfully reserved your Appchain ID via the Tanssi dApp.](/images/builders/deploy-manage/dapp/deploy/deploy-8.png)
+
+## Generate Your Appchain Files {: #generate-appchain-files }
+
+Before you can deploy your Appchain, you'll need to generate four configuration files:
+
+- [The raw chain specification](/builders/build/substrate/customizing-chain-specs/#generating-raw-specs-file){target=_blank} - a compact version of the JSON specification file, which defines the initial settings and state that all nodes participating in the network must agree on to reach consensus and produce blocks
+- [The genesis state](/builders/build/substrate/customizing-chain-specs/#genesis-state){target=_blank} - defines the initial state upon which all transactions and state transitions are executed
+- [The genesis Wasm](/learn/framework/architecture/#runtime){target=_blank} - a WebAssembly (Wasm) blob that defines the runtime validation logic
+- [The Tanssi genesis state](/builders/build/substrate/customizing-chain-specs/#genesis-state){target=_blank} - defines the genesis state for Tanssi
```

Comment:
@augustocollado added a brief description of each, please review this specifically

Reply:
- a WebAssembly (Wasm) blob that defines the runtime validation logic -> a WebAssembly (Wasm) blob that defines the runtime logic
- defines the genesis state for Tanssi -> defines the genesis state to register in Tanssi